programs/zstdcli.c

```diff
@@ -1411,25 +1411,33 @@ int main(int argCount, const char* argv[])
           assert(ZSTD_NB_STRATEGIES == strategyBounds.upperBound);
           (void)strategyBounds; }
 
-        if (showDefaultCParams) {
+        if (showDefaultCParams || g_displayLevel >= 4) {
             size_t fileNb;
             for (fileNb = 0; fileNb < (size_t)filenames->tableSize; fileNb++) {
                 unsigned long long fileSize = UTIL_getFileSize(filenames->fileNames[fileNb]);
                 const size_t dictSize = dictFileName != NULL ? (size_t)UTIL_getFileSize(dictFileName) : 0;
                 const ZSTD_compressionParameters cParams = ZSTD_getCParams(cLevel, fileSize, dictSize);
```

Comment:
This will give you the _default_ compression parameters.
But if any of these parameters is altered an advanced command line parameters, this won't be captured by this invocation, and therefore won't be faithfully reported.